scripts/components/dependencies_validator.ts

```diff
@@ -243,20 +251,31 @@ export class DependenciesValidator {
   private getPackageVersionDeclarationPredicate = async (
     packageName: string
   ): Promise<DependencyVersionPredicate> => {
-    if (packageName === 'execa') {
+    const inconsistentDependency =
+      this.knownInconsistentDependencyVersions.find(
+        (x) => x.dependencyName === packageName
+      );
+    if (inconsistentDependency) {
```

Comment:
it might be a good idea to move the comment section below to `check_dependnecies.ts`

Reply:
working on it now 👍